### ISSUE TITLE
Issues with comp() in v0.5.0

### ISSUE THREAD

**Issue by onesandzeroes** [dardisco/survMisc#8]
When I try to run comp() on a simple example using `survMisc` 0.5.0, I get strange output with negative test statistics etc.:

```r
> library(survMisc)
> fit = survfit(Surv(time, status) ~ x, data = aml)
> comp(ten(fit))
                   Q       Var       Z    pNorm  
1           -3.68934   4.04897 -1.8335 0.066731 .
n          -50.00000 925.00000 -1.6440 0.100178  
sqrtN      -12.83792  55.81480 -1.7184 0.085727 .
S1          -2.10453   1.64882 -1.6390 0.101222  
S2          -1.95449   1.45827 -1.6185 0.105553  
FH_p=1_q=1  -0.45686   0.14587 -1.1962 0.231616  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
             maxAbsZ       Var      Q  pSupBr
1            3.68934   4.04897 1.8335 0.13346
n           50.00000 925.00000 1.6440 0.20035
sqrtN       12.83792  55.81480 1.7184 0.17145
S1           2.10453   1.64882 1.6390 0.20244
S2           1.95449   1.45827 1.6185 0.21110
FH_p=1_q=1   0.45686   0.14587 1.1962 0.46257
```

However, reverting back to v0.4.6 these tests work fine:

```r
> library(survMisc)
> fit = survfit(Surv(time, status) ~ x, data = aml)
> comp(fit)$tests$lrTests
                                 ChiSq df       p
Log-rank                      3.396389  1 0.06534
Gehan-Breslow (mod~ Wilcoxon) 2.723312  1 0.09889
Tarone-Ware                   2.981604  1 0.08422
Peto-Peto                     2.708035  1 0.09984
Mod~ Peto-Peto (Andersen)     2.640333  1 0.10418
Flem~-Harr~ with p=1, q=1     1.452483  1 0.22813
```

Session info (using v0.5.0, but all other packages should be the same):

```
R version 3.2.3 (2015-12-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 15.10

locale:
 [1] LC_CTYPE=en_AU.UTF-8       LC_NUMERIC=C               LC_TIME=en_AU.UTF-8        LC_COLLATE=en_AU.UTF-8     LC_MONETARY=en_AU.UTF-8   
 [6] LC_MESSAGES=en_AU.UTF-8    LC_PAPER=en_AU.UTF-8       LC_NAME=C                  LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_AU.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] survMisc_0.5.0  survival_2.38-3

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.3      lattice_0.20-33  zoo_1.7-12       chron_2.3-47     grid_3.2.3       plyr_1.8.3       gtable_0.2.0     KMsurv_0.1-5    
 [9] scales_0.4.0     ggplot2_2.1.0    data.table_1.9.6 splines_3.2.3    tools_3.2.3      munsell_0.4.3    colorspace_1.2-6 knitr_1.12.3    
[17] km.ci_0.5-2      gridExtra_2.2.1
```

Happy to add any other information that's needed, just let me know.

**Comment by dardisco** [comment]
Thanks! V. odd. I'm sure it's the packageVersion that's at fault. Thanks also for providing a minimal working example to test.
Will try fix ASAP.

**Comment by drcanak** [comment]
Right,

I'm seeing the same behavior.  I have a very simple data set I use for class, that I am willing to share.  The results were definitely not right, as (a) I had test statistics in the 100s, and (b) results were not the same when cross-checked against SPSS.  Reverting back to 0.4.6 worked.  I'm also happy to provide any information that might help you troubleshoot the issue(s).

Regardless, thx for the package.  My students very much like to see alignment between what SPSS provides as output and what R provides.  To that end, your "comp" function is most appreciated.

**Comment by dardisco** [comment]
Sorry for delay!
I believe this has been fixed in Version 0.5.2, which is out on CRAN today (windows binaries should be built shortly). I was using the wrong 'observed minus expected' column...
I don't have a copy of SPSS to check this against, but the results are the same as Version 0.4.6 and the examples appear to duplicate textbook results where relevant.
Thanks to you both for pointing this out.
